src/test/java/me/gnat008/perworldinventory/ReflectionTestUtils.java

```diff
@@ -0,0 +1,34 @@
+package me.gnat008.perworldinventory;
+
+import java.lang.reflect.Field;
+
+import static java.lang.String.format;
+
+/**
+ * Offers reflection functionality to set up tests. Use only when absolutely necessary.
+ */
+public final class ReflectionTestUtils {
```

Comment:
TestHelper also has some reflection methods, please move/merge those into this new class